ksim/task/rl.py

```diff
@@ -298,16 +294,26 @@ def train_loop(
 
             # Updates the model on the collected trajectories.
             with self.step_context("update_state"):
-                params, opt_state = self.model_update(
+                params, opt_state, loss_val, metrics = self.model_update(
                     model, params, optimizer, opt_state, trajectories
                 )
 
-            # # Logs the trajectory statistics.
+            self.curr_logging_data = LoggingData(
+                trajectory=trajectories,
+                update_metrics=metrics,
+                gradients=None,
+                loss=float(loss_val),
+                training_state=training_state,
+            )
+
+            # Logs the trajectory statistics.
             with self.step_context("write_logs"):
-                #     state.phase = "train"
-                #     self.log_state_timers(state)
-                #     self.log_trajectory_stats(env, trajectories)
-                #     self.logger.write(state)
+                training_state.raw_phase = "train"
+                for log_item in self.log_items:
+                    self.logger.log_scalar(
+                        log_item.get_name(), lambda logger=log_item: logger(self.curr_logging_data), namespace="📉"
```

Comment:
nit: add comma at the end, then format. should result in something like this
```suggestion
                        log_item.get_name(),
                        lambda logger=log_item: logger(self.curr_logging_data),
                        namespace="📉",
```

Reply:
nice thank you